quiz/quiz.css

```diff
@@ -0,0 +1,180 @@
+@import url('https://vaspacx-components.netlify.app/style.css');
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@200;300;400;500;600&display=swap');
+/* @import url('https://vaspacx-login.netlify.app/login.css'); */
+:root{
+    --mainBrandColor: #3a7ce3;
+    --secondaryDark: #211c1c;
+    --font-primary: #FFFFFF;
+    --lightGrey: rgb(114 110 110);
+    --review-color: #ffb000;
+    --secbg-color: #211c1ca6;
+    --logo-color:white;
+    --url: url("../img/darkbg.jpg");
+    --quiznoColor:#ff72bf;
+    --btnquizColor: #191b7d;
+} 
+
+.light-theme{
+    --mainBrandColor: #3a7ce3;
+   --secondaryDark: #eef0fd;
+   --font-primary: rgb(0 0 0);
+   --lightGrey: rgb(114 110 110);
+   --review-color: #ffb000;
+   --secbg-color: #e2e0ec52;
+   --logo-color:black;
+   --url: url("../img/lightbg.jpg");
+   --quiznoColor:#02c8f5;
+   --btnquizColor: #191b7d;
+}
+
+
+
+
+#icon{
+    width:30px;
+    cursor:pointer;
+    position: absolute;
+    top:5%;
+    right:5%;
+}
+
+body{
+    background: var(--url);
+    color:var(--font-primary);
+    display:flex;
+    flex-direction:column;
+    align-items:center;
+    justify-content: center;
+    background-size: 100vw 100vh;
+    background-repeat: no-repeat;
+    width: 100vw;
+    height: 100vh;
+}
+
+
+.quiz-container{
+    background-color: var(--secbg-color);
+    display: flex;
+    justify-content: center;
+    padding: 20px;
+    border-radius: 5px;
+    width: 760px;
+    height: 138px;
+    margin-bottom: 20px;
+    align-items: center;
+}
+
+.quiz-option-cont{
+    background-color: var(--secbg-color);
+    border-radius: 10px;
+    display: flex;
+    justify-content: start;
+    padding: 10px;
+    width: 383px;
+    height: 75px;
+    align-items: center;
+}
+
+.display-wrap{
+    display: flex;
+    flex-direction: column;
+    width: 50%;
```

Comment:
Instead of using width % property every time you can make col width helper class

look into these link - https://www.w3schools.com/css/tryit.asp?filename=tryresponsive_cols